.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: ci
+
+on: push
```

Comment:
I believe this is expected behavior when creating a PR from a fork - a `pull_request` event is being submitted, and the workflow in the base repo (`godaddy/tartufo`) is ran if it has a matching `on:` statement. Since there are no GitHub Action workflows defined yet in `godaddy/tartufo`, nothing is ran, in line with https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories

I added `pull_request` along with the push event, so that in the future, PR's will get built using the base repository workflow.